dev-requirements.txt

```diff
@@ -1,11 +1,11 @@
 # install latest changes in dbt-core
 # TODO: how to automate switching from develop to version branches?
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-tests-adapter&subdirectory=tests/adapter
+git+https://github.com/dbt-labs/dbt-core.git@feature/support-incremental-predicates#egg=dbt-core&subdirectory=core
+git+https://github.com/dbt-labs/dbt-core.git@feature/support-incremental-predicates#egg=dbt-tests-adapter&subdirectory=tests/adapter
 
 # install latest changes in dbt-spark
 # TODO: how to automate switching from develop to version branches?
-git+https://github.com/dbt-labs/dbt-spark.git#egg=dbt-spark
+git+https://github.com/dbt-labs/dbt-spark.git@feature/incremental-predicates#egg=dbt-spark
```

Comment:
Are these changes still necessary? If so, we need to wait for the changes in `dbt-core` and `dbt-spark`.

Reply:
for now, yes! hoping to merge the `dbt-core` and `dbt-spark` PRs shortly!